po/minetest.pot

```diff
@@ -218,7 +218,7 @@ msgid "Uninstall"
 msgstr ""
 
 #: builtin/mainmenu/dlg_contentstore.lua
-msgid "View"
+msgid "View more information in a web browser"
```

Comment:
Please revert this change, it will be done automatically by a script.

Reply:
Did that myself since I checkout out your branch to run `optipng` anyway.

Reply:
Huh, I missed those two files, my bad